10.9-libcxx/stable/main/finkinfo/web/django-py.info

```diff
@@ -16,32 +16,55 @@ BuildDepends: setuptools-tng-py%type_pkg[python]
 #  mysql-python-py%type_pkg[python]
 Recommends: psycopg2-py%type_pkg[python], yaml-py%type_pkg[python]
 
-#Source: https://pypi.python.org/packages/source/D/Django/Django-%v.tar.gz
-Source: https://www.djangoproject.com/m/releases/1.11/Django-%v.tar.gz
-Source-MD5: ebdac613143ebdca911d5cef326fdc53
+#Source: https://files.pythonhosted.org/packages/source/D/Django/Django-%v.tar.gz
+Source: https://www.djangoproject.com/m/releases/2.1/Django-%v.tar.gz
+Source-Checksum: SHA256(d6393918da830530a9516bbbcbf7f1214c3d733738779f06b0f649f49cc698c3)
 
 DocFiles: AUTHORS INSTALL LICENSE README.rst docs
 
 PatchScript: <<
-  #!/bin/bash -ev
+  #!/bin/sh -ev
   perl -pi -e "s|'geos_c'|'libgeos_c.1.dylib'|g" django/contrib/gis/geos/libgeos.py
   perl -pi -e "s|lib_path = None|lib_path = '%p/opt/libgeos%type_raw[geos]/lib/libgeos_c.1.dylib'|g" django/contrib/gis/geos/libgeos.py
   perl -pi -e "s|lib_path = None|lib_path = '%p/lib/libgdal.1.dylib'|g" django/contrib/gis/gdal/libgdal.py
 <<
 
 CompileScript: <<
-   echo Skipping compile stage
+  %p/bin/python%type_raw[python] setup.py build
 <<
 
 InstallScript: <<
-   #!/bin/bash -ev
-   %p/bin/python%type_raw[python] setup.py install --root=%d
-   mkdir -p %i/share/doc/%n
-   cp -pr docs %i/share/doc/%n
-   install -d %i/etc/bash_completion.d
-   install extras/django_bash_completion %i/etc/bash_completion.d/django-py%type_pkg[python]
-   mv %i/bin/django-admin.py %i/bin/django-admin.py%type_raw[python]
-   mv %i/bin/django-admin %i/bin/django-admin%type_raw[python]
+  #!/bin/sh -ev
+  %p/bin/python%type_raw[python] setup.py install --root=%d
+  mkdir -p %i/share/doc/%n
+  cp -pr docs %i/share/doc/%n
+  install -d %i/etc/bash_completion.d
+  install extras/django_bash_completion %i/etc/bash_completion.d/django-py%type_pkg[python]
+  mv %i/bin/django-admin.py %i/bin/django-admin.py%type_raw[python]
+  mv %i/bin/django-admin %i/bin/django-admin%type_raw[python]
+<<
+
+InfoTest: <<
+  TestDepends: <<
+    bcrypt-py%type_pkg[python],
+    docutils-py%type_pkg[python],
+    jinja2-py%type_pkg[python] ( >= 2.9.2),
+    numpy-py%type_pkg[python],
+    pil-py%type_pkg[python] (>= 5.4.1),
+    pytz-py%type_pkg[python],
+    yaml-py%type_pkg[python]
+  <<
+  TestScript: <<
+    #!/bin/sh -ev
+    cd tests
+    # Run tests in parallel on all available cores (default) - set to 1 for better diagnostics
+    # ~25 ascii/unicode errors for Python < 3.7
+    if [ %type_pkg[python] -lt 37 ]; then
+      PYTHONPATH=%b/build/lib %p/bin/python%type_raw[python] -B runtests.py --parallel=0 || exit 1
```

Comment:
Setting `LANG="en_US.UTF-8" ` for the py36 and below tests makes them pass for me (fixes the ascii encoding failure).